metric_config_parser/tests/integration/test_config_integration.py

```diff
@@ -93,9 +93,11 @@ def test_config_from_repo_tree_multiple(self):
         assert len(config_collection.definitions) > 0
 
     def test_config_collection_from_branch(self):
-        config_collection = ConfigCollection.from_github_repo(
-            "https://github.com/mozilla/metric-hub/tree/main",
-            "https://github.com/mozilla/metric-hub/tree/main/jetstream/",
```

Comment:
Was this broken before? It seems like it shouldn't have worked properly looking at the `from_github_repo` function.